core/app/views/layouts/refinery/admin.html.erb

```diff
@@ -2,22 +2,30 @@
 <%= render '/refinery/html_tag' %>
   <% content_for :meta, tag(:meta, name: 'refinerycms', content: Refinery.version) %>
   <%= render 'refinery/admin/head' %>
-  <body class="<%= action_name %> <%= I18n.locale %>">
+  <body class="<%= action_name %> <%= I18n.locale %> cms-interface">
     <%= render 'refinery/admin/menu' %>
     <%= render '/refinery/site_bar' %>
     <div id='tooltip_container'></div>
-    <div id="admin_container" class="clearfix">
-      <div id="page_container">
-        <div id="page">
-          <div id="content" class="clearfix">
-            <div id="flash_container">
-              <%= render '/refinery/no_script' %>
-              <%= render '/refinery/message' %>
+    <section id="wrapper" class="dimmer">
+      <% if content_for?(:admin_content) %>
```

Comment:
Here's what I'm thinking

```
pick 0dc3d38 Allow for admin index views
s b80ebdc A temporary search partial --> a modification of 0dc3d38
s 643d583 media queries, rems over px's --> a modification of 0dc3d38
s 9c4db16 rm tmp_search partial. Coming soon in refinerycms-search --> revert b80ebdc
s 5fbaeef rm search styles as they belong in refinerycms-search --> a modification of 0dc3d38
s 126bca9 a nice light gray background --> forgot to include this in 0dc3d38
s 416ed28 rm styles for index with nested rows (a tree) --> a modification of 0dc3d38
```

Which of these would you like to see as its own commit?  I think 643d583 is the only possibility, but it is really just a modification of 0dc3d38.  126bca9 is sort of a feature, but I always intended it to be in the first commit.